extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -423,6 +494,7 @@ class CornerstoneViewportService implements IViewportService {
       displaySetInstanceUIDs.push(displaySetInstanceUID);
 
       if (!volume) {
+        console.log('Not adding volume');
```

Comment:
better log message please, this is just three words and will get basically hidden in the pile of log messages we have